C# extension sometimes fails to extract OmniSharp

There seems to be some sort of race/non-deterministic issue with the way OmniSharp is downloaded and extracted. The result of this bug are that, after installing the C# extension, there are no IntelliSense features, and if one opens up the OmniSharp output log, the following is shown:

    [INFO] Starting OmniSharp at '/home/greggm/clicon'...
    [ERROR] Error: Could not fnd launch file in /home/greggm/.vscode-alpha/extensions/ms-vscode.csharp-1.0.3-rc2/.omnisharp. Expected 'run', 'OmniSharp', 'OmniSharp'

If one does a directory listing, one will then find that there is only a .tmp file in the .omnisharp directory

    $ cd ~/.vscode/extensions/ms-vscode.csharp-1.*/.omnisharp
    $ ls
    tmp-4197TmzM6GRTyvfd.tmp

### Work arounds
If one restarts VS Code, the extension will try and download OmniSharp again. If you are unlucky, it will fail this second time too. But eventually the bad luck will end and OmniSharp will be there. Once it is there the bug goes away.

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "postinstall": "tsc"
   },
   "dependencies": {
-    "decompress": "^3.0.0",
+    "decompress": "^4.0.0",
     "del": "^2.0.2",
     "fs-extra-promise": "^0.3.1",
     "http-proxy-agent": "^1.0.0",

